packages/node_modules/@webex/widget-speed-dial/src/SpeedDialForm.tsx

```diff
@@ -262,7 +263,10 @@ export const SpeedDialForm = ({
                   name={value}
                   defaultValue={formData?.givenName}
                   autoFocus
+                  autoComplete="off"
                   label={t('form.firstName.label')}
+                  aria-label={t('form.firstName.placeholder')}
```

Comment:
Could you please assign aria-label as formData?.givenName instead of aria-label={t('form.firstName.placeholder')}?

Reply:
Sure, Akula